src/Core/OrganizationFeatures/OrganizationUsers/DeleteGroupCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Bit.Core.Exceptions;
```

Comment:
I only just noticed this is under `OrganizationUsers`, but Groups are a different feature.

I suggest creating a new `OrganizationFeatures/Groups` folder.

Reply:
Makes sense! I've made that change.